File: ru.kmorozov.library.data.model/src/main/java/ru/kmorozov/library/data/model/dto/CategoryDTO.java
package ru.kmorozov.library.data.model.dto;

import ru.kmorozov.library.data.model.book.Category;

/**
 * Created by sbt-morozov-kv on 04.04.2017.
 */
public class CategoryDTO {

    private String id;
    private String name;

    public CategoryDTO() {
    }

    public CategoryDTO(Category category) {
        id = category.getId();
        name = category.getName();
    }

    public String getId() {
        return this.id;
    }

    public String getName() {
        return this.name;
    }
}

File: ru.kmorozov.library.data.model/src/main/java/ru/kmorozov/library/data/model/dto/CategoryDTO.kt
package ru.kmorozov.library.data.model.dto

import ru.kmorozov.library.data.model.book.Category

/**
 * Created by sbt-morozov-kv on 04.04.2017.
 */
class CategoryDTO {

    lateinit var id: String
    lateinit var name: String

    constructor() {}

    constructor(category: Category) {
        this.id = category.id
        this.name = category.name
    }
}
